lib/PicoDb/Driver/Mssql.php

```diff
@@ -46,6 +46,9 @@ public function createConnection(array $settings)
         if (! empty($settings['port'])) {
             $dsn .= ';port=' . $settings['port'];
         }
+        if (! empty($settings['trust_server_cert'])) {
+            $dsn .= ';TrustServerCertificate=' . $settings['trust_server_cert'];
+        }
```

Comment:
Could we make this setting a boolean instead? Passing the string representation of the bool seems a little clunky.

E.g.

```php
if (isset($settings['trust_server_cert'])) {
    $dsn .= ';TrustServerCertificate=' . ($settings['trust_server_cert'] ? 'true' : 'false');
}
```